### PR TITLE
Describe Topiary configuration in README.md

### PR DESCRIPTION
**Pull request by ErinvanderVeen**
Thanks to @torhovland for reporting that this description was missing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +222,62 @@ It will output the following formatted code:
...
 
+## Configuration
+Topiary is configured using `languages.toml` files. There are three
+locations where Topiary checks for such a file.
+
+### Locations
+At buildtime the [languages.toml](./languages.toml) in the root of
+this repository is included into Topiary. This file is parsed at
+runtime. The purpose of this `languages.toml` file is to provide sane
+defaults for users of Topiary (both the library and the binairy).
```
Comment (314eter) [inline README.md:233]: binary

File: README.md [hunk 2]
```diff
@@ -222,6 +222,62 @@ It will output the following formatted code:
...
+
+The Topiary binary parses these file in the following order, any configuration
+options defined earlier are overwritten by those defined later.
+
+1. The builtin configuration file
+2. The user configuration file in the OS's configuration directory
+3. The project specific topiary configuration
+
+### Configuration Options
+The configuration file is contains a list of languages, each language
```
Comment (torhovland) [inline README.md:258]:
```suggestion
The configuration file contains a list of languages, each language
```

File: README.md [hunk 3]
```diff
@@ -222,6 +222,62 @@ It will output the following formatted code:
...
+configuration headed by the ``[[language]]`` toml header. For instance, the one
```
Comment (torhovland) [inline README.md:259]: I don't particularly like "headed by the header" 🙂

**Review by torhovland — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ErinvanderVeen) [reply README.md:259]: That's fair  :) Lemme fix that

**Pull request merged**